source/features/add-gists-link-to-profile.js

```diff
@@ -0,0 +1,16 @@
+import {h} from 'dom-chef';
+import select from 'select-dom';
+import {getCleanPathname} from '../libs/page-detect';
+import api from '../libs/api';
+
+export default async () => {
+	const username = getCleanPathname();
+	const {public_gists: publicGists} = await api(`users/${username}`);
+
+	select('.UnderlineNav-body').append(
+		<a href={`https://gist.github.com/${username}`} class="UnderlineNav-item" role="tab" title="Gists">
+			{'Gists '}
+			<span class="Counter">{publicGists}</span>
+		</a>
+	);
```

Comment:
Can you split this in two parts?

First add the tab, then fetch and add the counter.